src/hotspot/share/opto/cfgnode.cpp

```diff
@@ -1567,7 +1573,7 @@ static Node *is_x2logic( PhaseGVN *phase, PhiNode *phi, int true_path ) {
   // Build int->bool conversion
   Node *n = new Conv2BNode(cmp->in(1));
   if( flipped )
-    n = new XorINode( phase->transform(n), phase->intcon(1) );
+  n = new XorINode( phase->transform(n), phase->intcon(1) );
```

Comment:
This lives under the `if (flipped)`, maybe move into a block for more clarity.

Reply:
Fixed, thanks.